velox/type/Timestamp.h

```diff
@@ -151,14 +151,11 @@ struct Timestamp {
       const Precision& precision = Precision::kNanoseconds) const {
     // mbasmanova: error: no matching function for call to 'gmtime_r'
```

Comment:
do we need this comment what is it about @mbasmanova

Reply:
This comment can be removed. It says that seconds_ is long long, but `gmtime_r` does not have a function where the input parameter is long long, so it will report a 'no matching function' error. But no matter `gmtime` or `gmtime_r`, there is no matching function to input long long.
Besides, we still keep the following comment `// mbasmanova: time_t is long not long long` to draw attention.